lang/lang.de.json

```diff
@@ -0,0 +1,927 @@
+                "color": "{{REF:colors.success}}"
+            },
+            "removedSettingUser": {
+                "description": "Deine Einstellung **{{SETTING_NAME}}** wurde entfernt.",
+                "color": "{{REF:colors.success}}"
+            },
+            "removedGuild": {
+                "description": "Alle Daten von diesem Server wurden gelöscht.",
+                "color": "{{REF:colors.success}}"
+            },
+            "removedBot": {
+                "description": "Alle Daten zu {{BOT}} wurden gelöscht.",
+                "color": "{{REF:colors.success}}"
+            },
+            "removedUser": {
+                "description": "Deine Daten wurden gelöscht.",
+                "color": "{{REF:colors.success}}"
+            },
+            "addedListItem": {
+                "description": "**{{TIME_ZONE}}** wurde zur Liste der Zeitzonen hinzugefügt.",
+                "color": "{{REF:colors.success}}"
+            },
+            "removedListItem": {
+                "description": "**{{TIME_ZONE}}** wurde von der Liste der Zeitzonen entfernt.",
+                "color": "{{REF:colors.success}}"
+            }
+        },
+        "validation": {
+            "serverOnlyCommand": {
+                "description": "Dieser Befehl kann nur in einem Server ausgeführt werden.",
+                "color": "{{REF:colors.warning}}"
+            },
+            "permissionRequired": {
+                "description": "Du hast nicht die benötigten Berechtigungen, um diesen Befehl auszuführen!",
+                "color": "{{REF:colors.warning}}"
+            },
+            "notFoundSetting": {
+                "description": "Einstellung nicht gefunden.",
+                "color": "{{REF:colors.error}}"
+            },
+            "notFoundBot": {
+                "description": [
+                    "Bot nicht gefunden oder keine Einrichtung durchgeführt.",
+                    "",
+                    "Schreibe `{{REF:bot.prefix}} {{REF:commands.set}} <{{REF:arguments.bot}}>` um einen neuen Bot einzurichten."
+                ],
+                "color": "{{REF:colors.error}}"
+            },
+            "notFoundUser": {
+                "description": "Nutzer nicht gefunden.",
+                "color": "{{REF:colors.error}}"
+            },
+            "invalidTimeZone": {
+                "description": "Ungültige Zeitzone, bitte neu probieren!",
+                "color": "{{REF:colors.error}}"
+            },
+            "invalidDateFormat": {
+                "description": "Ungültiges Datumsformat, bitte neu probieren!",
+                "color": "{{REF:colors.error}}"
+            },
+            "invalidTimeFormat": {
+                "description": "Ungültiges Zeitformat, bitte neu probieren!",
+                "color": "{{REF:colors.error}}"
+            },
+            "invalidYesNo": {
+                "description": "Ungültige "ja"-oder "nein"-Antwort, bitte neu probieren!",
```

Comment:
This line should look like this:
`"description": "Ungültige \"ja\"-oder \"nein\"-Antwort, bitte neu probieren!"`

Otherwise that breaks JSON formatting.